src/Elements/Serialization/IFC/IFCModelExtensions.cs

```diff
@@ -146,7 +147,6 @@ internal static void ToIFC(Model model, string path)
             // IfcRelAssociatesMaterial
             // IfcMaterialDefinitionRepresentation
             // https://forums.buildingsmart.org/t/where-and-how-will-my-colors-be-saved-in-ifc/1806/12
-            var materials = new Dictionary<string, IfcMaterial>();
```

Comment:
This is the fix for #253. This dictionary is not required. Removing it fixes the error where adding two materials of the same name would cause a duplicate key exception.